torch/distributions.py

```diff
@@ -62,12 +68,17 @@ class Bernoulli(Distribution):
     Args:
         probs (Tensor or Variable): the probabilty of sampling `1`
     """
+
     def __init__(self, probs):
         self.probs = probs
 
     def sample(self):
         return torch.bernoulli(self.probs)
 
+    def sample_n(self, n):
+        expanded_probs = self.probs.unsqueeze(0).expand(n, *self.probs.size())
```

Comment:
same comment here, not sure the `unsqueeze` is necessary and can be done with `self.probs.expand(n, *self.probs.size())`